### ISSUE TITLE
Freeze on initial connection

### ISSUE THREAD

**Issue by wreckedcarzz** [Olrosse/BeamMP-Outbreak#1]
Heya! I'm the 'dev' for the ToW fork of Outbreak, but I'm just learning about Lua and beam scripting.

The official BeamNG update, and subsequent BeamMP update, that was released today, has broken Outbreak. Attached is the beginning of the console error printout. I have read that the MP mod is moving away from TCP/IP for communication with the game itself (information from the release notes of the early access release of MP), but am unaware if they have rolled out those changes or if that is the cause of the error; but the error complains of TCP scripting, so I thought I'd give my thoughts.

To reproduce, simply start a server with no mods except Outbreak, join it, and look at the console.

I'd attempt to fix this myself but this is way above my knowledge; really hope it can be resolved :)

![image](https://github.com/Olrosse/BeamMP-Outbreak/assets/74814173/7fa3429a-02d6-416b-9435-c3dfe9ea4041)
![image](https://github.com/Olrosse/BeamMP-Outbreak/assets/74814173/4ad41cd4-1c61-4db2-8a72-587fe14dd157)

**Comment by Olrosse** [comment]
oh yes i'm aware of this, it is happening on line 347 in lua/ge/extensions/outbreak.lua or requestGameState(), it's when it's requesting the game state when the extension is loaded, it doesn't need that line so you can remove it, i thought i pushed that already but apparently i forgot,

i didn't realize there was a fork, i'll have to check it out soon :)

**Comment by wreckedcarzz** [comment]
Commenting out that line appears to have fixed it, fantastic! :D

I forked it since I initially could not figure out how to do a pull request to the Outbreak repo (not new to github but new as a contributor), and then each day I added more and more small changes, and while I think I understand now how to do a pull request, I feel like my changes would be a bit much and so I decided to keep them separate.

Since I have you, an unrelated question: I have spent the last week trying to get a debug nametag on certain players of my server (myself, admins, mods) during *non*games/matches, as we are running this on public servers and I want to stand out and be more accessible and approachable to other players, especially with issues, questions, or concerns; and I feel that I am *right there* at making it work, but I think I am not getting the correct nick/name variable (or something very similarly mistaken), and I'm pulling my hair out. Could I DM you somewhere for help?
